### DISCUSSION TITLE
v2.4.1-Beta

### DISCUSSION THREAD

**Discussion by Gkiokan** [Gkiokan/ps4-remote-pkg-sender#5] [Show and tell]
Added errorCatcher for better debugging for everyone.

Fixed the missing ps4 default port for the RPI which caused no Install Requests. 
Sorry for the mistake, that happend to me by rushing the configs and couldn't see the error on my dev machine until feedback comes in.

Again, if you have any trouble on running the portable, then use the unpacked version.

<hr /><em>This discussion was created from the release <a href='https://github.com/Gkiokan/ps4-remote-pkg-sender/releases/tag/v2.4.1'>v2.4.1-Beta</a>.</em>

**Comment by HardRom** [comment]
![image](https://user-images.githubusercontent.com/277477/147977875-1c00d6ad-b2e5-460f-b445-f0a0fe4df3ff.png)
On the new beta 2.4.1 portable and zip
Windows 11, running as admin- happens when I click on "Server".

Still not sending pkgs to PS4, timeout.

Hopefully this is helpful to debug?

**Reply by HardRom** [reply]
FYI- my path is a mapped network location, in case that matters.

**Reply by Gkiokan** [reply]
That is caused by the errorCatcher, it catches for now everything. But the ResizeObserver is just a hint, not an error. Thats fine.

Regarding your timeout. I got also timeouts on windows which seems to wait for the RPI response that may take sometimes.

In 2.4.1 is a fix buildin timeout of 2200ms. I could fix it by restarting the RPI on my PS4 and it worked. 
It may have something to do with the RPI that maybe prepares the Install process? Install request is sometimes over 1,8secs until response. But wondering that happens only on windows so far.

Your path can be everything as far your Operating System have access to it.

**Reply by HardRom** [reply]
Thanks for the reply @Gkiokan, I have restarted the rpi pkg each time (from your repo, originally replacing the one from homebrewstore), and the portable on load shows this without anything in the background (Application and Info, nothing else): 
![image](https://user-images.githubusercontent.com/277477/148124516-fd2941c1-d3cd-4a7c-9a29-3fcd12fbb328.png)

The extracted zip version 2.4.1 has the same, but the window in the background is populated:
![image](https://user-images.githubusercontent.com/277477/148124670-b95cb7c9-5272-48e6-bc5d-1cbca57f2a19.png)

The extracted one continues, but the "hint" appears after clicking OK, Server.

Queuing a pkg I've checked and installed previously time's out on clicking start,  I've rebooted my PC, am running your app as Admin, I've rebooted the PS4 and restarted the RPI between each attempt.

Please let me know what more I can assist to troubleshoot this with you.

**Reply by Gkiokan** [reply]
I've added a [Troubleshooting Guide](https://github.com/Gkiokan/ps4-remote-pkg-sender/blob/master/Troubleshoot.md)

Can you try the new release [v2.4.2](https://github.com/Gkiokan/ps4-remote-pkg-sender/releases/tag/v2.4.2)? There you can set your timeout value yourself.
If everything is correct you should Restart the RPI on your PS4. Helped me and some ppl for now. 

The Error Window is interesting. Noone had that before. I don't have win11 yet but I will try to search for that error in the codebase.

**Comment by AmirHAskari** [comment]
I cant send files it gives me Error 2157510789. please help.

**Reply by Gkiokan** [reply]
This is the Error Code from the Remote Package Installer on the PS4. But this one is completely new. 

Did you try another PKG to send? Known Error values are patched in the next patch (2157510681, 2157510663, 2157510677).

Which file are you trying to send? Can you try another PKG?

**Reply by AmirHAskari** [reply]
My PS4 was running out of storage and I didn't notice that. I deleted some games and now it works fine. Thank you.

**Reply by Gkiokan** [reply]
Thanks also for your help. This Error Code means then no storage.
